### DISCUSSION TITLE
可以支持geolite2的库么

### DISCUSSION THREAD

**Discussion by pmkol** [IrineSistiana/mosdns#429] [Q&A]
https://github.com/Hackl0us/GeoIP2-CN
看了眼，貌似这个库会更精准一些，不知道能否支持他的格式

**Comment by IrineSistiana** [comment]
反问: mosdns 支持哪些 IP 格式。

**Reply by pmkol** [reply]
> 反问: mosdns 支持哪些 IP 格式。

他的格式是mmdb，看到wiki文档里，貌似支持的是.dat格式的，所以不知道是否支持.mmdb的库

**Reply by IrineSistiana** [reply]
用cidr不就行了。

**Reply by pmkol** [reply]
cidr快8MB了，尺寸有点感人 - . -

**Reply by IrineSistiana** [reply]
这库有毒吧。不 merge 一下 cidr 么....
merge 完了和ipip的一样 6000 多行，多了约40行， 100KB.....

**Reply by pmkol** [reply]
> 这库有毒吧。不 merge 一下 cidr 么.... merge 完了和ipip的一样 6000 多行，多了约40行， 100KB.....

之前不知道有merge这种神器，直接自己搞了个规则库
https://github.com/pmkol/easymosdns/tree/rules
感谢作者大神 : D
